docs/entities/attributes.md

```diff
@@ -0,0 +1,59 @@
+| `KNOCKBACK_RESISTANCE`        | how strong knockback is reduced. Ranged between 0 and 1, 1 cancelling all knockback.         |
+| `MOVEMENT_SPEED`              | how fast the entity moves                                                                    |
+| `FLYING_SPEED`                | The flying speed of the entity. Not applicable to players who fly with the elytra            |
+| `ATTACK_DAMAGE`               | how much damage the entity deals when attacking                                              |
+| `ATTACK_KNOCKBACK`            | how much knockback the entity causes when physically attacking                               |
+| `ATTACK_SPEED`                | how many full damage attack the entity deals per second                                      |
+| `ARMOR`                       | how many armor points the entity has and how resistant it is to attacks                      |
+| `ARMOR_TOUGHNESS`             | how resistant the entity is to strong attacks                                                |
+| `LUCK`                        | changes the chance for treasure, junk and normal fishing loottables                          |
+| `SPAWN_REINFORCEMENTS_CHANCE` | how likely a zombie spawns reinforcements when attacked                                      |
+| `JUMP_STRENGTH`               | how high the entity jumps. Only applicable for horse like more (including mules and camels). |
+
+NeoForge adds the following attributes:
+
+| Attribute              | Description                                                            |
+|------------------------|------------------------------------------------------------------------|
+| `SWIM_SPEED`           | how fast the entity swims                                              |
+| `NAMETAG_DISTANCE`     | how far the nametag can be seen for this entity                        |
+| `ENTITY_GRAVITY`       | how fast the entity accelerates when not on ground                     |
+| `BLOCK_REACH`          | how far the player can interact with the world                         |
+| `ENTITY_REACH`         | how far the player can interact with entities                          |
+| `STEP_HEIGHT_ADDITION` | how high the entity can walk up blocks in addition to Entity#maxUpStep |
+
+Attribute, AttributeInstance and AttributeModifier
+-----------
+
+Attributes have to be [registered][registration] and are used to look up the current `AttributeInstance` for a given LivingEntity. The `AttributeInstance` holds all `AttributeModifier` currently active on the entity.
+AttributeModifiers require a UUID that should be unique to this modifier. It's used to ensure that an `AttributeModifier` can only be applied once to a LivingEntity. You should generate a random one once and then hardcode it. They also require an Operation and an amount to perform the calculation. How the calculation works can be read in the next section.
```

Comment:
`Operation` can be explained in a subcategory here.

Reply:
I have an apt description of Operation here: <https://github.com/Shadows-of-Fire/Apothic-Attributes/blob/1.20/src/main/java/dev/shadowsoffire/attributeslib/api/AttributeHelper.java#L52-L63>